plugins/kvscheduler/api/kv_scheduler_api.go

```diff
@@ -250,6 +250,16 @@ type KVScheduler interface {
 	// GetRecordedTransaction returns record of a transaction referenced
 	// by the sequence number.
 	GetRecordedTransaction(SeqNum uint64) (txn *RecordedTxn)
+
+	// ValidateSemantically validates given proto messages according to semantic validation(KVDescriptor.Validate)
+	// from registered KVDescriptors. If all messages are valid, nil is returned. If all message could be
+	// validated, kvscheduler.InvalidMessagesError is returned. In any other case, error is returned.
```

Comment:
> If all message could be validated, kvscheduler.InvalidMessagesError is returned.

Probably mistake in the comment.

Reply:
Fixed in [`695d734`](https://github.com/ligato/vpp-agent/pull/1786/commits/695d73462eb3acdba5360595e11c51f78ac7153f)